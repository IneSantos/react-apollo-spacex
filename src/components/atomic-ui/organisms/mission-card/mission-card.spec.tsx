import React from "react";
import { render } from "@testing-library/react";
import MissionCard from "./mission-card";

describe("MissionCard organism tests", () => {
  test(`MissionCard component render`, () => {
    const { container } = render(<MissionCard name="This is a mission name" details="This is a mission detail" link="" img=""/>);
    expect(container.firstChild).toMatchSnapshot();
  });
});
