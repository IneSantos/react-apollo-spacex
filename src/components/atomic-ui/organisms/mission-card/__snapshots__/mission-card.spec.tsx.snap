// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MissionCard organism tests MissionCard component render 1`] = `
<div
  class="mission-card-wrapper"
>
  <img
    alt="Launch image"
    src=""
  />
  <div
    class="card-header"
  >
    <h2
      class="title"
    >
      This is a mission name
    </h2>
  </div>
  <div
    class="card-body"
  >
    <div
      class="details"
    >
      This is a mission detail
    </div>
    <a
      class="card-action "
      href=""
    >
       
      See more
       
    </a>
  </div>
</div>
`;
